specification/workloadmonitor/resource-manager/Microsoft.WorkloadMonitor/preview/2020-01-13-preview/Microsoft.WorkloadMonitor.json

```diff
@@ -328,124 +328,126 @@
       "name": "subscriptionId",
       "required": true,
       "type": "string",
-      "description": "The subscriptionId of the resource",
+      "description": "The subscription Id of the virtual machine.",
       "x-ms-parameter-location": "method"
     },
     "resourceGroupName": {
       "in": "path",
       "name": "resourceGroupName",
       "required": true,
       "type": "string",
-      "description": "The resourceGroupName of the resource",
+      "description": "The resource group of the virtual machine.",
       "x-ms-parameter-location": "method"
     },
-    "resourceNamespace": {
+    "providerName": {
       "in": "path",
-      "name": "resourceNamespace",
+      "name": "providerName",
       "required": true,
       "type": "string",
-      "description": "The resourceNamespace of the resource",
+      "description": "The provider name (ex: Microsoft.Compute for virtual machines).",
       "x-ms-parameter-location": "method"
     },
-    "resourceType": {
+    "resourceCollectionName": {
       "in": "path",
-      "name": "resourceType",
+      "name": "resourceCollectionName",
       "required": true,
       "type": "string",
-      "description": "The resourceType of the resource",
+      "description": "The resource collection name (ex: virtualMachines for virtual machines).",
```

Comment:
Is the intention that this only applies to tracked resources, or can it apply to sub-resources as well?  Is there a possibility this value could contain a '/'?